lib/PuppeteerSharp/PageCoverage/CSSCoverage.cs

```diff
@@ -37,8 +42,11 @@ internal Task StartAsync(CoverageStartOptions options)
 
             _resetOnNavigation = options.ResetOnNavigation;
             _enabled = true;
-            _stylesheetURLs.Clear();
-            _stylesheetSources.Clear();
+            lock (_stylesheetsLock)
```

Comment:
Added this because async CDP changes from this PR means access to these dictionaries is no longer guaranteed to be thread-safe.

Reply:
Can't we have a concurrent dictionary?